client/src/components/DataFiles/DataFilesTable/DataFilesTable.js

```diff
@@ -23,13 +23,26 @@ const DataFilesTablePlaceholder = ({ section, data }) => {
   );
   const scheme = useSelector(state => state.files.params.FilesListing.scheme);
   const system = useSelector(state => state.pushKeys.target);
+  const systemString = useSelector(
+    state => state.files.params.FilesListing.system
+  );
   const loading = useSelector(state => state.files.loading[section]);
   const err = useSelector(state => state.files.error[section]);
   const modalRefs = useSelector(state => state.files.refs);
 
   const filesLength = data.length;
   const isGDrive = currentListing === 'googledrive';
 
+  useEffect(() => {
+    dispatch({ type: 'GET_SYSTEM_MONITOR' });
+  }, [dispatch]);
+  const downSysList = useSelector(state =>
+    state.systemMonitor
+      ? state.systemMonitor.list
+          .filter(currSystem => !currSystem.is_operational)
+          .map(downSys => downSys.hostname.split('.')[0])
```

Comment:
@edmondsgarrett  are you asking why `cloud.corral.tacc.utexas.edu` isn't  one of the entries in the system monitor ?

Reply:
No, I was just confused on which part of state I was looking for, and had found the incorrect one with my first guess.

Reply:
I don't think I am seeing how this will work with using the systems definitions list - as I understand, this list does not indicate which data files location the user is in currently, so knowing when to render the error message should the host be down seems convoluted.

Reply:
Each system in the list has an associated system id, so you can find the current listing host by mapping against the system id found in the FilesListing object

Reply:
Just pushed commit reflecting this logic, thanks for walking me through this thoroughly.